scripts/common/addon.sh

```diff
@@ -231,3 +231,49 @@ function addon_set_has_been_applied() {
         kubectl patch configmaps -n kurl kurl-current-config --type merge -p "{\"data\":{\"addons-$name\":\"$current\"}}"
     fi
 }
+
+# check if the files are already present - if they are, use that
+# if they are not, prompt the user to provide them
+# if the user does not provide the files, return 1
+function addon_fetch_airgap() {
+    local name=$1
+    local version=$2
+    local package="$name-$version.tar.gz"
+
+    if [ -f "$(package_filepath "${package}")" ]; then
+        # the package already exists, no need to download it
+        printf "The package %s %s is already available locally.\n" "$name" "$version"
+    else
+        # prompt the user to give us the package
+        printf "The package %s %s is not available locally, and is required.\n" "$name" "$version"
+        printf "You can download it from %s with the following command:\n" "$(get_dist_url)/${package}"
+        printf "\n${GREEN}    curl -o %s %s${NC}\n\n" "${package}" "$(get_dist_url)/${package}"
```

Comment:
```suggestion
        printf "\n${GREEN}    curl -LO %s${NC}\n\n" "$(get_dist_url)/${package}"
```

since in addon_fetch_nocache we use

```
curl -LO "$url"
```